### PR TITLE
Fix advent season feast day naming

### PR DESCRIPTION
**Pull request by codename-abbott**
#10 
* make Christmas Day reference point for calculating weeks of Advent

### CODE REVIEW

**Comment by codename-abbott** [comment]
@darksmo, I was not sure if to include the generated file from the js folder. Do I need to add that or is this enough? In case it is needed, I will add before sending this PR into review.
Thanks for the help!

File: test/catholicHolidays.js [hunk 1]
```diff
@@ -16,9 +16,10 @@ describe("getEasterSunday", function () {
 
 describe("getFirstSundayOfAdvent", function () {
     it("works as expected", function () {
+        expect(oHolidays.getFirstSundayOfAdvent(2021).format("DD-MM-YYYY")).to.equal("28-11-2021");
+        expect(oHolidays.getFirstSundayOfAdvent(2020).format("DD-MM-YYYY")).to.equal("29-11-2020");
         expect(oHolidays.getFirstSundayOfAdvent(2018).format("DD-MM-YYYY")).to.equal("02-12-2018");
         expect(oHolidays.getFirstSundayOfAdvent(2017).format("DD-MM-YYYY")).to.equal("03-12-2017");
-        expect(oHolidays.getFirstSundayOfAdvent(1996).format("DD-MM-YYYY")).to.equal("01-12-1996");
```
Comment (darksmo) [inline test/catholicHolidays.js:21]: Why was this removed?

**Review by darksmo — APPROVED** [review]
Change overall is good  - I'm merging it and adding that test back as it's passing with your change in.

**Comment by darksmo** [comment]
> @darksmo, I was not sure if to include the generated file from the js folder. Do I need to add that or is this enough? In case it is needed, I will add before sending this PR into review.
> Thanks for the help!

Was not needed - I commit all generated files when running the build.
